joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -1199,7 +1198,7 @@ class AstCreator(filename: String, global: Global)
       .name(name)
       .methodFullName(name)
       .signature(localIdentifier.getText())
-      .typeFullName(MethodFullNames.UnknownFullName)
+      .typeFullName(DynamicCallUnknownFullName)
```

Comment:
in order to be consistent with a frontend like _pythonsrc_, I'd change the TYPE_FULL_NAME for call like these to _"ANY"_